wakuv2/waku.go

```diff
@@ -1267,6 +1160,7 @@ func (w *Waku) Start() error {
 			case c := <-w.connStatusChan:
 				w.connStatusMu.Lock()
 				latestConnStatus := formatConnStatus(w.node, c)
+				w.logger.Info("PeerStats", zap.Any("stats", latestConnStatus))
```

Comment:
This might add too much noise to the logs, consider using debug instead

Reply:
Fixed.